### DISCUSSION TITLE
Password protection

### DISCUSSION THREAD

**Discussion by SSoft7** [FlareSolverr/FlareSolverr#671] [Ideas]
Hello,

I can see that flaresolverr has no authentication/password protection mechanism built-in. If I install this on a server, then anyone with the port number can make tons of request to flaresolverr making to server machine unusable which is somewhat similar to a DDoS attack.

Any plans to add inbuilt authentication? I think if you just add a "basic auth" that will be sufficient and also other apps can use it without much issue.

**Comment by 0xlane** [comment]
Agree

**Comment by ngosang** [comment]
https://github.com/FlareSolverr/FlareSolverr/issues/367
